Modules/mmapmodule.c

```diff
@@ -227,6 +227,14 @@ do {                                                                    \
     return err;                                                         \
     }                                                                   \
 } while (0)
+#define CHECK_VALID_AND_RELEASE(err, buffer)                            \
```

Comment:
Maybe `CHECK_VALID_OR_RELEASE`?

Reply:
Okay, it's really a more appropriate name for this macro.
I